cmd/kubeapps-apis/plugins/kapp_controller/packages/v1alpha1/server.go

```diff
@@ -40,14 +40,18 @@ import (
 
 const (
 	// See https://carvel.dev/kapp-controller/docs/latest/packaging/#package-cr
-	packageGroup     = "package.carvel.dev"
+	packageGroup     = "packaging.carvel.dev"
 	packageVersion   = "v1alpha1"
-	packagesResource = "packages"
+	packageResource  = "PackageInstall"
+	packagesResource = "packageinstalls"
 
 	// See https://carvel.dev/kapp-controller/docs/latest/packaging/#packagerepository-cr
-	installPackageGroup   = "install.package.carvel.dev"
+	installPackageGroup   = "packaging.carvel.dev"
```

Comment:
We should no longer need the separate `installPackageGroup` and `installPackageVersion`? These were only required because previously the CRs were on a different group.